### ISSUE TITLE
Mob TODO list

### ISSUE THREAD

**Issue by Wuzzy2** [maikerumine/mobs_mc#7]
Requirements to tick off a mob off the list:
* All requirements of #9 (done)
* Correct mob behaviour, including, but not limited to:
    * Attacking algorithm
    * Pathfinding y/n
    * Breeding
    * Feeding
* All required player interactions
* Correct model, textures and animations
* Sounds (#79)
* Bug-free
* Relevant code file is cleaned up
* Playtested

**Not** required is the implementation of easter eggs (feel free to open a new issue).

---

Most important enemies:
* [ ] Zombie
* [ ] Husk
* [ ] Skeleton
* [ ] Stray 
* [ ] Creeper 
* [ ] Slime 
* [ ] Spider
* [ ] Cave Spider 
* [ ] Enderman 
* [ ] Zombie Villager
* [ ] Zombie Pigman

Most important passive mobs:
* [ ] Chicken
* [ ] Cow
* [ ] Pig 
* [ ] Rabbit 
* [ ] Sheep
* [ ] Squid 
* [ ] Polar Bear 
* [ ] Bat 
* [ ] Mooshroom

Horses:
* [ ] Horse
* [ ] Donkey 
* [ ] Llama 
* [ ] Mule 
* [ ] Skeleton Horse
* [ ] Zombie Horse (PE)

Complex “helper” mobs:
* [ ] Wolf
* [ ] Villager
* [ ] Iron golem
* [ ] Snow golem
* [ ] Ocelot 
* [ ] Parrot 

Location-based enemies:
* [ ] Wither Skeleton
* [ ] Magma Cube 
* [ ] Blaze
* [ ] Ghast 
* [ ] Evoker 
* [ ] Vex 
* [ ] Vindicator 
* [ ] Witch 
* [ ] Guardian 
* [ ] Silverfish 
* [ ] Endermite 
* [ ] Shulker

Bosses:
* [ ] Ender Dragon 
* [ ] Wither 
* [ ] Elder Guardian (PE)

**Comment by maikerumine** [comment]
THis is a very good list.  Do we check them off as we complete?

**Comment by Wuzzy2** [comment]
Yes, of course. But at the moment, none of the mobs so far meets the (arguably strict) criteria at the moment. I wouldn't bother about this issue yet and instead focus on #9.

**Comment by kingoscargames** [comment]
The "Very rare mobs" aren't different mobs, they are mobs riding another mob.

**Comment by Wuzzy2** [comment]
Mob mounts have been moved to new issue #73.
